src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -550,6 +561,55 @@ private static bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 			return true;
 		}
 
+		private bool ProcessCssFile(IRequestInfo info, string localPath)
+		{
+			// is this request the full path to a real file?
+			if (File.Exists(localPath) && Path.IsPathRooted(localPath))
+			{
+
+				// currently this only applies to languageDisplay.css, settingsCollectionStyles.css, and customCollectionStyles.css
+				var cssFile = Path.GetFileName(localPath);
+				if ((cssFile == "languageDisplay.css") || (cssFile == "settingsCollectionStyles.css") || (cssFile == "customCollectionStyles.css"))
+				{
+					info.ContentType = "text/css";
+					info.ReplyWithFileContent(localPath);
+					return true;
+				}
+			}
+
+			// if not a full path, try to find the correct file
+			var fileName = localPath;
+			var pos = fileName.LastIndexOfAny(new[] { '\\', '/' });
+			if (pos > -1)
+				fileName = fileName.Substring(pos + 1);
+
+			// try to find the css file in the xmatter and templates
+			var path = (_fileLocator == null) ? string.Empty : _fileLocator.LocateFile(fileName);
```

Comment:
The settingsCollectionStyles.css and customCollectionStyles.css files are referenced as "../settingsCollectionStyles.css" in the HTML, so the browser requests them from the project directory rather than the book directory.